### DISCUSSION TITLE
Up_date Release  since 2.4.72 not ok

### DISCUSSION THREAD

**Discussion by f5exo** [Bodmer/TFT_eSPI#2285] [Raspberry Pi Pico]
Hello 
I'm using TFT_eSPI release  **2.4.72**  with   Rapsberry Pico and TFT3.5 & ILI9488_DRIVER .
**Everything is ok and very well with this release, even Rapsberry Pico W.**
For new release Screen is white !
I used the Tests and Diagnostics exemples:
Read_User_Setup to check configuration.
All is ok.
So I have to keep this old version.
It's not blocking for me.

I don't know if this issue has been addressed before.
I can do some tests if you want...
Best regard.

setup below:

#include <User_Setups/Setup60_RP2040_ILI9341.h>    // Setup file for RP2040 with SPI ILI9341
#define ILI9488_DRIVER     // WARNING: Do not connect ILI9488 display SDO to MISO if other devices share the SPI bus (TFT SDO does NOT tristate when CS is high)

SPI configuration is:
// For the Pico use these #define lines
#define TFT_MISO 16
#define TFT_MOSI 19
#define TFT_SCLK 18
#define TFT_CS 17 // Chip select control pin
#define TFT_DC 10 // Data Command control pin
#define TFT_RST -1 // Reset pin (could connect to Arduino RESET pin)
//#define TFT_BL // LED backlight

**Comment by Bodmer** [comment]
I have not seen this problem. Which board package are you using?

**Comment by f5exo** [comment]
Happy New Year for you !

Thank's for your replay.

Raspberry Pi RP2040 Core platform file By Earle F. Philhower
name=Raspberry Pi RP2040 Boards(2.3.3)
version=2.3.3

**Reply by f5exo** [reply]
Its good !!
I just upgraded to 2.7.0 RP2240 board 
I'm sorry , I would have started with that.
Many thank's for all.

**Reply by Bodmer** [reply]
Glad it is working!
